src/internal/pjs/server.go

```diff
@@ -0,0 +1,18 @@
+package pjs
+
+import (
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage"
+	txnenv "github.com/pachyderm/pachyderm/v2/src/internal/transactionenv"
+	pjsserver "github.com/pachyderm/pachyderm/v2/src/pjs"
+)
+
+type Env struct {
+	DB      *pachsql.DB
+	TxnEnv  *txnenv.TransactionEnv
+	Storage *storage.Server
```

Comment:
Remove for now.  I'm not sure if we will have to pull this back in.  I think we should only need `fileset.Pin`, no the entire storage layer.  Either way we shouldn't have unused dependencies.